modules/sdk-coin-avaxp/test/unit/avaxp.ts

```diff
@@ -460,6 +481,34 @@ describe('Avaxp', function () {
       isTransactionVerified.should.equal(true);
     });
 
+    // TODO(CR-1073): find unsiged, signed and half signed transactions for AddPermissionlessValidator
+    it('should succeed to verify signed add permissionless validator transaction', async () => {
```

Comment:
these look duplicate, we need to find equivalent data for AddPermissionLessValidator transaction

Reply:
Yes correct, will address in the follow up ticket.